### ISSUE TITLE
Apache2 and uWSGI

### ISSUE THREAD

**Issue by brenard** [unbit/vpn-ws#5]
Hello,

I try to install vpn-ws server using Apache2 (on Debian Jessie) but it's does not work. I'm using mod_uswgi with a simple configuration :

    <Location /vpn>
        SetHandler uwsgi-handler
        uWSGISocket /run/vpn.sock
    </Location>

With this configuration, my client can connect to the server :

**Comment by brenard** [comment]
Anyone have an idea about this problem ? I still search a solution, by adding debug on vpn-ws code, using tcpdump to watch traffic between hosts and applications and I'm able to say that is Apache witch bloc traffic between vpn-ws client and server. I also try to use undocumented Apache mod_Ruwsgi module and it's does not work.

Any ideas are welcome !

**Comment by unbit** [comment]
Have you checked if some kind of traffic reached the vpn-ws daemon when behind apache ?

**Comment by brenard** [comment]
Yes I checked : I have traffic between Apache and vpn-ws daemon until HTTP Upgrade during WS handshake. After that step, I have nothing behind Apache but a could see request and answer between browser and Apache.
